src/locale/locales/ru/messages.po

```diff
@@ -0,0 +1,6232 @@
+msgstr "Отменить"
+
+#: src/view/com/modals/CreateOrEditList.tsx:349
+#: src/view/com/modals/DeleteAccount.tsx:174
+#: src/view/com/modals/DeleteAccount.tsx:296
+msgctxt "action"
+msgid "Cancel"
+msgstr "Отменить"
+
+#: src/view/com/modals/DeleteAccount.tsx:170
+#: src/view/com/modals/DeleteAccount.tsx:292
+msgid "Cancel account deletion"
+msgstr "Отменить удаление учетной записи"
+
+#: src/view/com/modals/ChangeHandle.tsx:144
+msgid "Cancel change handle"
+msgstr "Отменить изменение псевдонима"
+
+#: src/view/com/modals/crop-image/CropImage.web.tsx:159
+msgid "Cancel image crop"
+msgstr "Отменить обрезку изображения"
+
+#: src/view/com/modals/EditProfile.tsx:245
+msgid "Cancel profile editing"
+msgstr "Отменить изменения профиля"
+
+#: src/view/com/util/post-ctrls/RepostButton.tsx:130
+msgid "Cancel quote post"
+msgstr "Отменить цитирование поста"
+
+#: src/screens/Deactivated.tsx:155
+msgid "Cancel reactivation and log out"
+msgstr "Отменить реактивацию и выйти из системы"
+
+#: src/view/com/modals/ListAddRemoveUsers.tsx:87
+#: src/view/shell/desktop/Search.tsx:214
+msgid "Cancel search"
+msgstr "Отменить поиск"
+
+#: src/view/com/modals/LinkWarning.tsx:106
+msgid "Cancels opening the linked website"
+msgstr "Отменяет открытие ссылки"
+
+#: src/view/com/modals/VerifyEmail.tsx:160
+msgid "Change"
+msgstr "Изменить"
+
+#: src/view/screens/Settings/index.tsx:371
+msgctxt "action"
+msgid "Change"
+msgstr "Изменить"
+
+#: src/view/screens/Settings/index.tsx:718
+msgid "Change handle"
+msgstr "Изменить псевдоним"
+
+#: src/view/com/modals/ChangeHandle.tsx:156
+#: src/view/screens/Settings/index.tsx:729
+msgid "Change Handle"
+msgstr "Изменить псевдоним"
+
+#: src/view/com/modals/VerifyEmail.tsx:155
+msgid "Change my email"
+msgstr "Изменить адрес электронной почты"
+
+#: src/view/screens/Settings/index.tsx:763
+msgid "Change password"
+msgstr "Изменить пароль"
+
+#: src/view/com/modals/ChangePassword.tsx:142
+#: src/view/screens/Settings/index.tsx:774
+msgid "Change Password"
+msgstr "Изменение пароля"
+
+#: src/view/com/composer/select-language/SuggestedLanguage.tsx:73
+msgid "Change post language to {0}"
+msgstr "Изменить язык поста на {0}"
+
+#: src/view/com/modals/ChangeEmail.tsx:104
+msgid "Change Your Email"
+msgstr "Изменить адрес электронной почты"
+
+#: src/Navigation.tsx:302
+#: src/view/shell/bottom-bar/BottomBar.tsx:201
+#: src/view/shell/desktop/LeftNav.tsx:295
+msgid "Chat"
+msgstr "Чат"
+
+#: src/components/dms/ConvoMenu.tsx:82
+msgid "Chat muted"
+msgstr "Чат заглушен"
+
+#: src/components/dms/ConvoMenu.tsx:112
+#: src/components/dms/MessageMenu.tsx:81
+#: src/Navigation.tsx:307
+#: src/screens/Messages/List/index.tsx:88
+#: src/view/screens/Settings/index.tsx:638
+msgid "Chat settings"
+msgstr "Настройки чата"
+
+#: src/screens/Messages/Settings.tsx:59
+#: src/view/screens/Settings/index.tsx:647
+msgid "Chat Settings"
+msgstr "Настройки чата"
+
+#: src/components/dms/ConvoMenu.tsx:84
+msgid "Chat unmuted"
+msgstr "Чат разглушен"
```

Comment:
We wanted to change that too, it's a strange wording.

Reply:
The thing is that this was regarding the issue of chat and user silencing, I had previously accepted a Pull Request with this translation. Thinking about it a bit now, I thought it would make sense to translate it as “Чат со звуком”